tests/modules/cmdintrospection.c

```diff
@@ -29,57 +29,61 @@ int RedisModule_OnLoad(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
-    RedisModuleCommandArg *trim_approx = RedisModule_CreateCommandArg("approx", REDISMODULE_ARG_TYPE_PURE_TOKEN, "~", NULL, NULL, REDISMODULE_CMD_ARG_NONE, NULL);
-    RedisModuleCommandArg *trim_op = RedisModule_CreateCommandArg("trim_op", REDISMODULE_ARG_TYPE_ONEOF, NULL, NULL, NULL, REDISMODULE_CMD_ARG_OPTIONAL, NULL);
+    RedisModuleCommandArg *trim_exact = RedisModule_CreateCommandArg("equal", REDISMODULE_ARG_TYPE_PURE_TOKEN, -1, "=", NULL, NULL, REDISMODULE_CMD_ARG_NONE, NULL);
+    RedisModuleCommandArg *trim_approx = RedisModule_CreateCommandArg("approximately", REDISMODULE_ARG_TYPE_PURE_TOKEN, -1, "~", NULL, NULL, REDISMODULE_CMD_ARG_NONE, NULL);
+    RedisModuleCommandArg *trim_op = RedisModule_CreateCommandArg("operator", REDISMODULE_ARG_TYPE_ONEOF, -1, NULL, NULL, NULL, REDISMODULE_CMD_ARG_OPTIONAL, NULL);
     RedisModule_AppendSubarg(trim_op, trim_exact);
     RedisModule_AppendSubarg(trim_op, trim_approx);
 
-    RedisModuleCommandArg *trim_threshold = RedisModule_CreateCommandArg("trim_threshold", REDISMODULE_ARG_TYPE_STRING, NULL, NULL, NULL, REDISMODULE_CMD_ARG_NONE, "threshold");
+    RedisModuleCommandArg *trim_threshold = RedisModule_CreateCommandArg("threshold", REDISMODULE_ARG_TYPE_STRING, -1, NULL, NULL, NULL, REDISMODULE_CMD_ARG_NONE, "threshold");
 
-    RedisModuleCommandArg *trim_count = RedisModule_CreateCommandArg("trim_count", REDISMODULE_ARG_TYPE_INTEGER, "LIMIT", NULL, "6.2.0", REDISMODULE_CMD_ARG_OPTIONAL, "count");
+    RedisModuleCommandArg *trim_count = RedisModule_CreateCommandArg("count", REDISMODULE_ARG_TYPE_INTEGER, -1, "LIMIT", NULL, "6.2", REDISMODULE_CMD_ARG_OPTIONAL, "count");
 
-    RedisModuleCommandArg *trimming = RedisModule_CreateCommandArg("trimming", REDISMODULE_ARG_TYPE_BLOCK, NULL, NULL, NULL, REDISMODULE_CMD_ARG_OPTIONAL, NULL);
+    RedisModuleCommandArg *trimming = RedisModule_CreateCommandArg("trim", REDISMODULE_ARG_TYPE_BLOCK, -1, NULL, NULL, NULL, REDISMODULE_CMD_ARG_OPTIONAL, NULL);
     RedisModule_AppendSubarg(trimming, trim_startegy);
     RedisModule_AppendSubarg(trimming, trim_op);
     RedisModule_AppendSubarg(trimming, trim_threshold);
     RedisModule_AppendSubarg(trimming, trim_count);
 
-    /* ID arg */
-    RedisModuleCommandArg *id_auto = RedisModule_CreateCommandArg("id_auto", REDISMODULE_ARG_TYPE_PURE_TOKEN, "*", NULL, NULL, REDISMODULE_CMD_ARG_NONE, NULL);
-    RedisModuleCommandArg *id_given = RedisModule_CreateCommandArg("id_given", REDISMODULE_ARG_TYPE_STRING, NULL, NULL, NULL, REDISMODULE_CMD_ARG_NONE, "id");
-    RedisModuleCommandArg *id = RedisModule_CreateCommandArg("id", REDISMODULE_ARG_TYPE_ONEOF, NULL, NULL, NULL, REDISMODULE_CMD_ARG_NONE, NULL);
+    // ID arg
+    RedisModuleCommandArg *id_auto = RedisModule_CreateCommandArg("auto_id", REDISMODULE_ARG_TYPE_PURE_TOKEN, -1, "*", NULL, NULL, REDISMODULE_CMD_ARG_NONE, NULL);
```

Comment:
just making sure you also updated the example in the docs with these changes

Reply:
yes i did

Reply:
actually i didn't but it doesn't really matter (I adjusted the code itself just to fit XADD's json, but it's just names of things...)

Reply:
i did add the `-1` where needed cause otherwise it wouldn't compile